### PR TITLE
removed empty py files

### PR DESCRIPTION
**Pull request by Ephibbs**

### CODE REVIEW

**Comment by Ephibbs** [comment]
solves #67

**Review by vmoens — CHANGES_REQUESTED** [review]
Once the `__init__.py` is edited and the checks pass we're good to go.

File: torchrl/record/rendering.py [hunk 1]
```diff
@@ -1,4 +0,0 @@
-# Copyright (c) Meta Platforms, Inc. and affiliates.
```
[the remark this answers is not in the document]
Comment (Ephibbs) [reply torchrl/record/rendering.py:1]: Yes, I updated this in the latest commit.

**Comment by vmoens** [comment]
LGTM thanks!

**Review by vmoens — APPROVED** [review]

**Pull request merged**